test/TestExtensions/GenerateLicenseList/GenerateLicenseList.csproj

```diff
@@ -7,7 +7,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
     <AssemblyName>GenerateLicenseList</AssemblyName>
-    <NoWarn>NU1505</NoWarn> <!-- Remove NoWarn when https://github.com/dotnet/sdk/issues/24747 is fixed -->
+    <NoWarn>$(NoWarn);NU1505</NoWarn> <!-- Remove NoWarn when https://github.com/dotnet/sdk/issues/24747 is fixed -->
```

Comment:
This can probably be removed altogether as the linked issue is likely fixed, but I understand if you don't want to risk it.

Reply:
Since it's blocking everyone, let's merge this without messing with other warnings.

Reply:
Yep, makes sense. 

Approval stands, feel free to merge.